pkg/sdkserver/sdkserver.go

```diff
@@ -474,40 +470,35 @@ func (s *SDKServer) Reserve(ctx context.Context, d *sdk.Duration) (*sdk.Empty, e
 
 	e := &sdk.Empty{}
 
-	s.logger.Info("Received Reserve request, adding to queue")
-	s.enqueueState(stablev1alpha1.GameServerStateReserved)
-
 	// 0 is forever.
-	if d.Seconds == 0 {
-		return e, nil
+	if d.Seconds > 0 {
```

Comment:
This check actually means that 0 _or a negative value_ means forever.

Reply:
Yeah, that was deliberate. I wanted to handle the error condition in a sane way.

Do you feel I should change the docs?